src/main/java/org/marek/model/User.java

```diff
@@ -0,0 +1,18 @@
+package org.marek.model;
+
+import javax.persistence.Entity;
+
+@Entity
+
+public class User {
+    private Integer id;
+    private String name;
+    private String surname;
+    private Integer phoneNumber;
```

Comment:
Bude povinné, proto zvolíme anotaci `@NotNull`. Také, telefonní číslo se vyjadřuje pomocí Stringu. Proč? Vždy, když se rozhodujeme mezi String a číslem, rozhoduje skutečnost, jestli budeme provádět aritmetické operace nad sloupcem nebo ne. S číslem nijak počítat nebudeme, takže to bude String. Telefonní číslo má taky nějakou maximální delku, takže použijeme `@Size(min=X, max=Y)`, kde X a Y jsou hranice, jak velké to číslo může být (kolik znaků). Tady tě nechám pogooglit, tohle je dost praktický příklad, který se hodí vědět :)